gui-libs/wlroots/wlroots-9999.ebuild

```
@@ -60,20 +65,20 @@ BDEPEND="
 "
 
 src_configure() {
+	local backends="$(usex drm 'drm,' '')"
```

You can simplify this a bit:
```suggestion
	local backends="$(usev drm 'drm,')"
```
etc

---

A great idea! Will be done

---

Done in https://github.com/gentoo/gentoo/commit/ad83b87f3f988a2cf276e4a2659c241c48b0bfb1